src/cascadia/WinRTUtils/inc/WtExeUtils.h

```diff
@@ -114,10 +116,8 @@ _TIL_INLINEPREFIX const std::wstring& GetWtExePath()
 // - arg - the command-line argument to quote and escape.
 // Return Value:
 // - the quoted and escaped command-line argument.
-_TIL_INLINEPREFIX std::wstring QuoteAndEscapeCommandlineArg(const std::wstring_view& arg)
+inline void QuoteAndEscapeCommandlineArg(const std::wstring_view& arg, std::wstring& out)

```

Comment:
Should this one also get `_TIL_INLINEPREFIX` so that it gets COMDAT-folded when it is multiply defined? Or is this fine?

Reply:
`_TIL_INLINEPREFIX` is not actually about inlining. It explicitly means _don't inline_. It's dumb, but...

It means "when this is seen in multiple .obj files, do not treat it as multiple definitions. Do COMDAT folding on it."

Reply:
Hmm... I may be misunderstanding what you're saying.
I thought `inline` on a function doesn't imply that it gets inlined, but instead that if multiple instances of it exist, the linker is allowed to deduplicate/fold them. I've assumed that the `__declspec(noinline) inline` trick says to never inline the function but to then also deduplicate it, which is required since it's in a header file. I can read what you wrote both as in support and in opposition to this, so if you don't mind: Could you clarify what you meant? 😅

Reply:
I was wondering if just using `inline` was safe for the linker, that's all. 😄

Reply:
(e.g. that it wasn't going to blow up with a linker error a year down the line when we include the file somewhere else) (but also, I know you cannot promise me that the linker will be safe)